src/include/pagestat_inline.h

```diff
@@ -6,20 +6,12 @@
  * See the file LICENSE for redistribution information.
  */
 
-#define WT_STAT_NONE -1
-
 /* Initialize the fields in a page stat structure to their defaults. */
-#define WT_PAGE_STAT_INIT(ps)            \
-    do {                                 \
-        (ps)->byte_count = WT_STAT_NONE; \
-        (ps)->row_count = WT_STAT_NONE;  \
+#define WT_PAGE_STAT_INIT(ps) \
+    do {                      \
+        (ps)->user_bytes = 0; \
```

Comment:
We still need a invalid value.

Reply:
@quchenhao Even if we have an invalid value we still need to store it if the other stat is present and valid. Is this what we want?

Reply:
Yes, that is correct